src/test/java/picard/metrics/CollectRrbsMetricsTest.java

```diff
@@ -1,102 +0,0 @@
-/*
```

Comment:
why is this test removed? we should still be testing CollectRrbsMetrics directly, not only via collect multiple metrics.